src/test/regress/expected/multi_limit_clause.out

```diff
@@ -519,5 +519,35 @@ SELECT
           1 |    1
 (1 row)
 
+-- non constants should not push down
+EXPLAIN (COSTS OFF) SELECT * FROM lineitem_hash LIMIT random()*10;
+                            QUERY PLAN
+---------------------------------------------------------------------
+ Limit
+   ->  Custom Scan (Citus Adaptive)
+         Task Count: 4
+         Tasks Shown: One of 4
+         ->  Task
+               Node: host=localhost port=xxxxx dbname=regression
+               ->  Seq Scan on lineitem_hash_360132 lineitem_hash
+(7 rows)
+
+EXPLAIN (COSTS OFF) SELECT * FROM lineitem_hash LIMIT 10 OFFSET random()*10;
+                            QUERY PLAN
+---------------------------------------------------------------------
+ Limit
+   ->  Custom Scan (Citus Adaptive)
+         Task Count: 4
+         Tasks Shown: One of 4
+         ->  Task
+               Node: host=localhost port=xxxxx dbname=regression
+               ->  Seq Scan on lineitem_hash_360132 lineitem_hash
```

Comment:
Here the test fails because of the randomness I guess.

Would it make sense to add a normalization rule for this line? So that it becomes:

```
Seq Scan on lineitem_hash_xxxxxxx lineitem_hash
```

Reply:
maybe, but I'm fixing these tests.